app/models/photo.rb

```diff
@@ -0,0 +1,29 @@
+# == Schema Information
+#
+# Table name: photos
+#
+#  id             :bigint           not null, primary key
+#  caption        :text
+#  comments_count :integer          default(0)
+#  image          :string
+#  likes_count    :integer          default(0)
+#  created_at     :datetime         not null
+#  updated_at     :datetime         not null
+#  owner_id       :bigint           not null
+#
+# Indexes
+#
+#  index_photos_on_owner_id  (owner_id)
+#
+# Foreign Keys
+#
+#  fk_rails_...  (owner_id => users.id)
+#
+class Photo < ApplicationRecord
+  belongs_to :owner, class_name: "User", counter_cache: true
```

Comment:
if in the Comment and Like model they have a belong to photo, then Photo model should have a hasMany

Reply:
Missed the has_many likes myself!